python/pyarrow/array.pxi

```diff
@@ -21,6 +21,8 @@ import os
 import warnings
 from cython import sizeof
 
+from pyarrow.compute import run_end_encode
```

Comment:
Instead of this import here (which isn't guaranteed to work, as one can build pyarrow without compute support), you can use `_pc()` to get the compute module where calling `run_end_encode` (see the other examples of that in this file)